packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -409,6 +410,8 @@ export default function DataviewsPatterns() {
 					isLoading={ isResolving }
 					view={ view }
 					onChangeView={ setView }
+					selection={ selection }
+					onChangeSelection={ setSelection }
```

Comment:
I don't understand why this fix is needed. I mean we're not doing anything with "selection" here. So for me, it means that "uncontrolled" selection in DataViews is not working properly and we should fix it within the component rather than switching to "controlled" selection.

Reply:
Hm.. sorry for missing this. I haven't worked much on the selection and didn't remember about uncontrolled version.. I'll take a look for the root issue then.